Tests/IntegrationTests/PostgresClientTests.swift

```diff
@@ -42,6 +42,41 @@ final class PostgresClientTests: XCTestCase {
             taskGroup.cancelAll()
         }
     }
+    
+    func testTransaction() async throws {
+        var mlogger = Logger(label: "test")
+        mlogger.logLevel = .debug
+        let logger = mlogger
+        let eventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: 8)
+        self.addTeardownBlock {
+            try await eventLoopGroup.shutdownGracefully()
+        }
+        
+        let clientConfig = PostgresClient.Configuration.makeTestConfiguration()
+        let client = PostgresClient(configuration: clientConfig, eventLoopGroup: eventLoopGroup, backgroundLogger: logger)
+        
+        await withThrowingTaskGroup(of: Void.self) { taskGroup in
+            taskGroup.addTask {
+                await client.run()
+            }
+            
+            let iterations = 1000
+            
+            for _ in 0..<iterations {
+                taskGroup.addTask {
```

Comment:
I think we should create a throwaway table here and insert some data. then we should check if a commit commits and if a rollback rolls back.

Reply:
> I think we should create a throwaway table here and insert some data. then we should check if a commit commits and if a rollback rolls back.

I added the tests in the latest commit. Thanks!